src/options.json

```diff
@@ -5852,6 +5852,24 @@
           "namespace": "php"
         }
       },
+      "allow_risky": {
```

Comment:
This should be automatically generated. I do not see a change to the PHP language adding `allow_risky` option?